core/src/main/resources/reference.conf

```diff
@@ -86,6 +86,10 @@ entityStatisticsCache {
   workspaceCooldown = 4 minutes
 }
 
+entities {
+  pageSizeLimit = 300000
+}
+
```

Comment:
if we want to change this later, we can copy this stanza into rawls.conf in firecloud-develop, instead of changing it here.